src/Preferences.cc

```diff
@@ -160,6 +160,7 @@ void Preferences::init() {
 	this->defaultmap["advanced/localization"] = true;
 	this->defaultmap["advanced/autoReloadRaise"] = false;
 	this->defaultmap["advanced/enableSoundNotification"] = true;
+	this->defaultmap["advanced/enableParameterCheck"] = true;
```

Comment:
One day we'll have to split up the "Advanced" tab, but that's a future adventure..

Reply:
good idea. At least grouping would be a good first step.
I will come back to this, likely when I add even more options related to errors and warnings.